clients/src/main/java/org/apache/kafka/common/requests/AddOffsetsToTxnResponse.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.requests;
+
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.apache.kafka.common.protocol.Errors;
+import org.apache.kafka.common.protocol.types.Struct;
+
+import java.nio.ByteBuffer;
+
+public class AddOffsetsToTxnResponse extends AbstractResponse {
+    private static final String ERROR_CODE_KEY_NAME = "error_code";
+
+    // Possible error codes:
+    //   NotCoordinator
+    //   CoordinatorNotAvailable
+    //   CoordinatorLoadInProgress
+    //   InvalidPidMapping
+    //   InvalidTxnState
+    //   GroupAuthorizationFailed
```

Comment:
As I am implementing the client, I am finding that sometimes the KIP is correct, and sometimes the code is correct, and it takes some careful consideration of the details to tell the difference. 

I vote to not iterate on this right now. We will fix the KIP and the code during the implementation, because that is the point where the details will be most clear.